Src/BootCamp.Chapter/Statics.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace BootCamp.Chapter
+{
+    internal class Statics
+    {
+        internal static void FindPostOfficeWithMostErrors(List<Address> data)
```

Comment:
If the above is PostOffice, then the method becomes:
PostOffice.FindMostErrors(data)